docs/contributor/testing-strategy.md

```diff
@@ -0,0 +1,95 @@
+# Testing Strategy Infrastructure-Manager
+
+## Introduction
+This testing strategy describes how the Framefrog team is testing the product `Kyma Infrastructure Manager`. It outlines the approach and methodologies to be used for testing all layers of this product to ensure the stability, reliability, and correctness.
+
+
+## Testing Methodology
+
+We investigate the product by separating it into layers:
+
+1. Business Logic
```

Comment:
What does the `business logic` mean? And how is it different than `Business Features`?